pkg/regionspan/span.go

```diff
@@ -231,15 +224,15 @@ func IsSubSpan(sub ComparableSpan, parents ...ComparableSpan) bool {
 	return false
 }
 
-// ToComparableSpan returns a memcomparable span
+// ToComparableSpan returns a comparable span.
```

Comment:
It's not a typo https://github.com/facebook/mysql-5.6/wiki/MyRocks-record-format

Reply:
Thanks!